kernel/build/config-amd64

```diff
@@ -3160,11 +3160,20 @@ CONFIG_ACPI_I2C_OPREGION=y
 CONFIG_I2C_BOARDINFO=y
 CONFIG_I2C_COMPAT=y
 # CONFIG_I2C_CHARDEV is not set
-# CONFIG_I2C_MUX is not set
+CONFIG_I2C_MUX=m
```

Comment:
is this needed for v4l?

Reply:
It's definitely not loaded on a system where this is running since none of the UVC drivers or it's dependents have any dependencies on I2C mux. It gets enabled with enabling multimedia in `make menuconfig`, but having a look at the Kconfig (https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/drivers/media/Kconfig?h=v6.1.69) `MEDIA_SUPPORT_FILTER` defaults to yes and `MEDIA_SUBDRV_AUTOSELECT` selects `I2C_MUX` and defaults to yes if `MEDIA_SUPPORT_FILTER`. 

```
config MEDIA_SUPPORT_FILTER
	bool "Filter media drivers"
	default y if !EXPERT
	help
	   Configuring the media subsystem can be complex, as there are
	   hundreds of drivers and other config options.

	   This menu offers option that will help the Kernel's config
	   system to hide drivers that are out of the scope of the
	   user needs, and disabling core support for unused APIs.

	   If not selected, all non-optional media core functionality
	   needed to support media drivers will be enabled. Also, all
	   media device drivers should be shown.

config MEDIA_SUBDRV_AUTOSELECT
	bool "Autoselect ancillary drivers (tuners, sensors, i2c, spi, frontends)"
	depends on HAS_IOMEM
	select I2C
	select I2C_MUX
	default y if MEDIA_SUPPORT_FILTER
	help
	  By default, a media driver auto-selects all possible ancillary
	  devices such as tuners, sensors, video encoders/decoders and
	  frontends, that are used by any of the supported devices.

	  This is generally the right thing to do, except when there
	  are strict constraints with regards to the kernel size,
	  like on embedded systems.

	  Use this option with care, as deselecting ancillary drivers which
	  are, in fact, necessary will result in the lack of the needed
	  functionality for your device (it may not tune or may not have
	  the needed demodulators).

	  If unsure say Y.
```


I guess I'll try with `MEDIA_SUPPORT_FILTER`,  `MEDIA_SUBDRV_AUTOSELECT`, and `I2C_MUX` disabled and make sure this is still functional.

Reply:
Removed in favor of a more minimal config in the latest push